Multisig/Presentation/UI Library/Buttons.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Buttons.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 14.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct GNOFilledButtonStyle: ButtonStyle {
```

Comment:
So basically instead of making custom views, you make custom styles?
Is it possible to do the same with custom views? Just out of curiosity :)

Reply:
This is the preferred way (as far as the system does button styles) for styling the buttons in SwiftUI. We could have a functions creating custom buttons, too.